lua/node.lua

```diff
@@ -15,4 +15,18 @@ node.input = function(str)
   )
 end
 
+node.chipid = function()
```

Comment:
shouldn't chipid be constant across one test case?
i'd suggest to mimic mdns implementation and move the variables (chipid and reason) and math.random to nodemcu-module re-assignable variables.

Reply:
Thanks! Seems like a better solution to me too I'll amend my PR.